src/pyxplor/pyxplor.py

```diff
@@ -0,0 +1,62 @@
+def plot_binary(input_df: pd.DataFrame,
+                list_of_variables: list,
+                orientation: str,
+                fig_width: int,
+                fig_height: int,
+                plot_kind: str,
+                label_y_offset: int,
+                label_fontsize: int,
+                output_path: str = None,
+                super_title: str = "Distribution of Binary Variables",
+                super_title_font: int = 14) -> None:
+    """Plot the distribution of binary variables in a DataFrame, save the plot, and display it.
+
+    This function will construct a set of subplots (either a bar plot or a pie chart)
+    for each of the binary variables specified in `list_of_variables`. The function will
+    save as well as display the entire figure. 
+
+    Parameters
+    ----------
+    input_df : pd.DataFrame
+        The DataFrame that contains the binary variables.
+
+    list_of_variables : list
+        List of binary variables (column names) to be plotted.
+
+    orientation : {'v', 'h'}
+        Orientation of the plots, either vertical or horizontal.
+
+    fig_width : int
+        Width of the figure.
+
+    fig_height : int
+        Height of the figure.
+
+    plot_kind : {'count', 'pie'}
+        Type of plot to be generated, a count plot or a pie chart.
+
+    label_y_offset : int
+        Y-axis offset for label positioning.
+
+    label_fontsize : int
+        Font size for axis labels.
+
+    output_path : str, optional
+        Path to save the plot. Defaults to the current working directory.
```

Comment:
Above it says default is None - we should pick either None or current working directory